src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -396,6 +459,10 @@ private function evaluate( $directive_value, string $default_namespace, $context
 			}
 		}
 
+		if ( $current instanceof Closure ) {
+			$current = $current();
```

Comment:
This seems like an easy place to break things by passing a closure that throws. Should we wrap this in try/catch with doing it wrong?

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/6394/commits/3f571c01b5e42a907ec527bb40f15847d297ac49.